test/sslapitest.c

```diff
@@ -6672,6 +6672,61 @@ static int test_ticket_callbacks(int tst)
     return testresult;
 }
 
+/*
+ * Test incorrect shutdown.
+ * Test 0: TLSv1.3, client does not shutdown properly,
+ *                  server does not set SSL_OP_IGNORE_UNEXPECTED_EOF,
+ *                  server should get SSL_ERROR_SSL
+ * Test 1: TLSv1.3, client does not shutdown properly,
```

Comment:
It says TLSv1.3 here - but actually you haven't specified any particular protocol version in the test below (the min/max protocol version parameters to `create_ssl_ctx_pair` are set to 0. So if TLSv1.3 is enabled we will use it. Otherwise we will use TLSv1.2 (or below of TLSv1.2  is also disabled). Should we test both TLSv1.2 *and* TLSv1.3? Or is one sufficient? Either way this comment isn't quite right.

Reply:
I think the version is irrelevant here. Updated the comment.